app/ldap.js

```diff
@@ -564,6 +564,9 @@ exports.addGroup = function (group, cb) {
   });
 };
 
+// check if LDAP is disabled
+exports.isDisabled = (!process.env.NODE_ENV || 'development' === process.env.NODE_ENV) && true !== process.env.LDAP;
```

Comment:
if I do: 
```bash
export NODE_ENV=development; export LDAP=true
```
```
node app/app 
```
What happens?  Add tests to confirm that this works. ALWAYS add tests -- I'm going to require tests for GSoC as well.